test/manager/bundler/artifacts.spec.ts

```diff
@@ -30,13 +30,7 @@ describe('bundler.updateArtifacts()', () => {
     };
 
     processEnv = process.env;
-    process.env = {
-      HTTP_PROXY: 'http://example.com',
-      HTTPS_PROXY: 'https://example.com',
-      NO_PROXY: 'localhost',
-      HOME: '/home/user',
-      PATH: '/tmp/path',
-    };
+    process.env = mockedEnv;
```

Comment:
shouldn't we simply mock `getChildProcessEnv` from `lib/util/env` ?

Reply:
Some code relies on `process.env` before doing `exec`

Reply:
So we should refactor all code directly referencing `process.env` to a single util function (seperate pr). Then we can test more easily.